src/django/api/serializers/v1/production_locations_serializer.py

```diff
@@ -1,36 +1,54 @@
 from rest_framework import serializers
```

Comment:
Could you please group imports from serializers?
Like:
from rest_framework.serializers import (
    IntegerField,
    FloatField,
    ListField,
    CharField,
    ChoiceField,
    Serializer,
    ValidationError,
)

Reply:
Done